src/main/java/teammates/common/util/Const.java

```diff
@@ -701,6 +701,9 @@ private Const() {
         public static final String FEEDBACK_SESSION_TIMEZONE = "timezone";
         public static final String FEEDBACK_SESSION_GRACEPERIOD = "graceperiod";
         public static final String FEEDBACK_SESSION_TYPE = "fstype";
+        public static final String FEEDBACK_SESSION_STANDARD = "STANDARD";
+        public static final String FEEDBACK_SESSION_TEAMEVALUATION = "TEAMEVALUATION";
+        public static final String FEEDBACK_SESSION_OPTIMIZEDTEAMEVALUATION = "OPTIMIZEDTEAMEVALUATION";
```

Comment:
Technically speaking "STANDARD" isn't a template - it just means using the instructor's own questions rather than a template :P 

Though I guess I can name `FEEDBACK_SESSION_TEMPLATE_TEAMEVALUATION` and `FEEDBACK_SESSION_TEMPLATE_OPTIMIZEDTEAMEVALUATION` instead...